examples/index.json

```
@@ -1433,6 +1433,13 @@
         "verifyOutput": true,
         "isLearnByExample": true
       },
+      {
+        "name": "Named worker with on fail",
+        "url": "named-worker-with-on-fail",
```

Let's include the word `clause` in the url and as well as in the BBE name similar to https://ballerina.io/learn/by-example/single-use-with-on-fail-clause/

```suggestion
        "url": "named-worker-with-on-fail-clause",
```